esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -591,7 +592,13 @@ def teachersbyFOO(self, request, tl, one, two, module, extra, prog, sort_exp = l
 
         for teacher in teachers:
             # get list of valid classes
-            classes = [ cls for cls in teacher.getTaughtSections(self.program)
+            if teaching and moderating:
+                class_objects = teacher.getTaughtSections(self.program) | teacher.getModeratingSectionsFromProgram(self.program)
```

Comment:
i think will made a combined moderating & teaching sections that i can replace this with